docs/explanation/examples.md

```diff
@@ -0,0 +1,16 @@
+# Examples
+
+You can find exhaustive examples how to use `pynxtools-mpes` for your ARPES research data pipeline [here](https://gitlab.mpcdf.mpg.de/nomad-lab/nomad-remote-tools-hub/-/tree/develop/docker/mpes).
```

Comment:
This will be moved here also eventually, no? For now, please update to the north repo.

Reply:
ok